### PR TITLE
Update the documentation of the GDAL extractprojection processing alg…

### LINKED ISSUE #3162: Backport of GDAL extractprojection reintroduction to 3.4 [needs-docs]
Original commit: https://github.com/qgis/QGIS/commit/97979e6ad4f3577e161473cd1f45226dc5105dc8 by web-flow

Backport of GDAL extractprojection reintroduction to 3.4 [needs-docs]

### PR DESCRIPTION
**Pull request by havatv**
…orithm including new output

Fixes #3162

### CODE REVIEW

File: source/docs/user_manual/processing_algs/gdal/rasterprojections.rst [hunk 1]
```diff
@@ -35,17 +35,20 @@ Parameters
 
 Extract projection
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/gdal/rasterprojections.rst]: Extract projection has been around for a long time in QGIS (version 2).  That it not has been available in QGIS 3 before 3.4 is a regression (that now has been fixed). So I am not sure about adding |34|.  I will, however, add |34| on the new outputs.

File: source/docs/user_manual/processing_algs/gdal/rasterprojections.rst [hunk 2]
```diff
@@ -35,17 +35,20 @@ Parameters
...
 ------------------
-Extracts the projection of a raster file and writes it into a
-world-file.
+Extracts the projection of a raster file and writes it into a `world`
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/gdal/rasterprojections.rst]: I think it needs to be emphasized - let us try with *world*.

File: source/docs/user_manual/processing_algs/gdal/rasterprojections.rst [hunk 3]
```diff
@@ -56,6 +59,12 @@ Parameters
 Outputs
 .......
 
+``World file`` [file]
+  Text file with extension :file:`.wld` containing transformation parameters
+  for  the raster file.
+
+``ESRI Shapefile prj file`` [file]
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/gdal/rasterprojections.rst]: PRJ_FILE will always be present in the results, but will have the value None if PRJ_FILE_CREATE is False.

File: source/docs/user_manual/processing_algs/gdal/rasterprojections.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (DelazJ) [reply source/docs/user_manual/processing_algs/gdal/rasterprojections.rst]: i think that 2.x users will refer to 2.18 docs which should have that alg described but 3.x users will read testing so 3.0 or 3.2 users that find that escription could think that they can access it and complain/be frustrated to not find it. Also adding this tag is kind of "you want it, then upgrade" 😄

File: source/docs/user_manual/processing_algs/gdal/rasterprojections.rst [hunk 4]
```diff
@@ -56,6 +59,12 @@ Parameters
...
+  Text file that describes the CRS.
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/gdal/rasterprojections.rst]: Yes.

File: source/docs/user_manual/processing_algs/gdal/rasterprojections.rst [hunk 5]
```diff
@@ -35,17 +35,20 @@ Parameters
 
 Extract projection
 ------------------
-Extracts the projection of a raster file and writes it into a
-world-file.
+Extracts the projection of a raster file and writes it into a `world`
+file with extension :file:`.wld`.
 The algorithm is derived from the `GDAL srsinfo utility <https://www.gdal.org/gdalsrsinfo.html>`_ .
 
 ``Default menu``: :menuselection:`Raster --> Projections`
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/gdal/rasterprojections.rst]: This has been missed.  It belongs there with Assign and Warp, so it should be fixed.  I will try to find out what needs to be done.

File: source/docs/user_manual/processing_algs/gdal/rasterprojections.rst [hunk 6]
```diff
@@ -56,6 +59,13 @@ Parameters
 Outputs
 .......
 
+``World file`` [file] |34|
+  Text file with extension :file:`.wld` containing transformation parameters
+  for  the raster file.
+
+``ESRI Shapefile prj file`` [file] |34|
+  Text file with :file:.prj extension that describes the CRS.
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/gdal/rasterprojections.rst]:
```suggestion
  Text file with :file:`.prj` extension that describes the CRS.
```

File: source/docs/user_manual/processing_algs/gdal/rasterprojections.rst [hunk 7]
```diff
@@ -33,19 +33,22 @@ Parameters
 
 .. _gdalextractprojection:
 
-Extract projection
-------------------
-Extracts the projection of a raster file and writes it into a
-world-file.
+Extract projection |34|
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/gdal/rasterprojections.rst]: You need to add a reference in the substitution list at the bottom of the file

File: source/docs/user_manual/processing_algs/gdal/rasterprojections.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/gdal/rasterprojections.rst]: Sorry, cut and paste error...

File: source/docs/user_manual/processing_algs/gdal/rasterprojections.rst (hunk 7 above)
Comment (havatv) [reply source/docs/user_manual/processing_algs/gdal/rasterprojections.rst]: Hopefully fixed now.

File: source/docs/user_manual/processing_algs/gdal/rasterprojections.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/gdal/rasterprojections.rst]: extractprojection is now back in the raster menu under projections.
(see https://github.com/qgis/QGIS/pull/8493 and https://github.com/qgis/QGIS/pull/8492)

**Comment by DelazJ** [comment]
Thanks

**Pull request merged**